Library/Homebrew/formulary.rb

```diff
@@ -4,7 +4,7 @@
 # The Formulary is responsible for creating instances of Formula.
 # It is not meant to be used directly from formulae.
 
-class Formulary
+module Formulary
```

Comment:
I think this should be a seperate PR. It's pretty hidden here and could have unintended side effects, so I think others should have a chance to be aware of this change.